.github/workflows/ci_static-checks.yaml

```
@@ -73,6 +73,9 @@ jobs:
             -   name: "Install dependencies"
                 run: composer update --no-interaction --no-scripts
 
+            -   name: "Warmup dev env cache for PHPStan and Psalm Symfony integrations"
+                run: bin/console cache:warmup --env=dev
+
```

```suggestion
            -   name: "Warmup cache"
                run: bin/console cache:warmup --env=dev
```

We don't need an explanation here :p.